docs/cloud.md

```diff
@@ -7,66 +7,64 @@
 
 - [Cloud Cluster Guide](#cloud-cluster-guide)
   - [Overview](#overview)
-  - [GCP Marketplace](#gcp-marketplace)
+  - [Cluster Toolkit](#cluster-toolkit)
   - [Terraform](#terraform)
-    - [Quickstart Examples](#quickstart-examples)
+  - [Quickstart Examples](#quickstart-examples)
 
 <!-- mdformat-toc end -->
 
 ## Overview
 
 This guide focuses on setting up a cloud [Slurm cluster](./glossary.md#slurm).
-With cloud, there are decisions that need to be made and certain considerations
-taken into account. This guide will cover them and their recommended solutions.
+in the cloud. Deploying in a cloud environment involves various decisions and
+considerations, which this guide will cover, along with recommended solutions.
 
-There are two deployment methods for cloud cluster management:
+There are two primary deployment methods for cloud cluster management:
```

Comment:
The only way is Cluster Toolkit, we don't have any terraform in slurm-gcp to support using it to deploy a cluster.